### PR TITLE
feat: 🍰 List All Groups

### LINKED ISSUE #5568: 🚀 [Feature] Have A List Of All Groups
## 🚀 Feature

Have a list of all groups.

### Additional Informations

We already have a page with `My groups` at `https://stage.ocelot.social/my-groups`.
Possible options I see at the moment: (to be supplemented)

- we can have a side menu with sub pages under
  - `https://stage.ocelot.social/groups/XXX`
    - `https://stage.ocelot.social/groups` (index)
    - `https://stage.ocelot.social/groups/my`
- we can have a page with a tabs `Groups` and `My groups` unter `https://stage.ocelot.social/groups`

### PR DESCRIPTION
**Pull request by Mogge**
## 🍰 Pullrequest
List all groups 

### Issues
- fixes #5568

### Todo

- [X] None

### CODE REVIEW

**Review by ulfgebhardt — APPROVED** [review]
Code wise looks good, tested locally - works

image

Edgecases habe ich nicht geprüft

File: webapp/pages/groups.vue [hunk 1]
```diff
@@ -0,0 +1,192 @@
+<template>
```
Comment (Tirokk) [inline webapp/pages/groups.vue:1]: Super cool code dear ! 😍

**Review by Tirokk — APPROVED** [review]
Very good done @Mogge !!! 😍

I approve 🚀🚀💫💫

File: webapp/pages/groups.vue (hunk 1 above)
Comment (Mogge) [reply webapp/pages/groups.vue:1]: Thank you for the flowers

**Pull request merged**